mlflow/tracking/_model_registry/utils.py

```diff
@@ -161,6 +162,9 @@ def _get_store_registry():
     for scheme in DATABASE_ENGINES:
         _model_registry_store_registry.register(scheme, _get_sqlalchemy_store)
 
+    for scheme in ["", "file"]:
+        _model_registry_store_registry.register(scheme, _get_file_store)
```

Comment:
`_get_file_store()` returns an instance of `mlflow.store.tracking.file_store.FileStore`, but we want to return an instance of `mlflow.store.model_registry.FileStore`